bar-trender-front-end/src/components/ApiLoginForm.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
import React from "react";
```